source/shared/core_stmt.cpp

```diff
@@ -526,39 +530,13 @@ void core_sqlsrv_bind_param( _Inout_ sqlsrv_stmt* stmt, _In_ SQLUSMALLINT param_
             break;
         case IS_STRING:
             {
+                if ( sql_type == SQL_DECIMAL || sql_type == SQL_NUMERIC ) {
+                    adjustInputPrecision( param_z, decimal_digits );
```

Comment:
Actually do we have to adjust input precision when AE is not enabled?
Besides, what's the likelihood the user will input a string of scientific notation for a decimal/numeric type?

Reply:
yes, if the user specifies the sqltype (only the case for SQLSRV driver) as SQL_DECIMAL or SQL_NUMERIC, the input passed into SQLBindParam requires matching scale (i.e., number of decimal digits).

I don't know what is the likelihood of user inputting a stringified scientific notation, but it's better to have this case covered.